pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/legacy/vedlegg/OpplysningerBruktIBeregningUTLegacy.kt

```diff
@@ -28,11 +27,12 @@ import no.nav.pensjon.brev.template.dsl.textExpr
 @TemplateModelHelpers
 val vedleggOpplysningerBruktIBeregningUTLegacy =
     createAttachment<LangBokmalNynorskEnglish, PE>(
-        title = newText(
-            Bokmal to "Opplysninger om beregningen",
-            Nynorsk to "Opplysningar om utrekninga",
-            English to "Information about calculations"
-        ),
+        title =
```

Comment:
Usikker på om jeg er enig i denne. Om det kun er for multi-line, så er det kanskje greit.